### DISCUSSION TITLE
期待Python后端~

### DISCUSSION THREAD

**Discussion by yqs112358** [Tencent/ScriptX#15] [General]
改开关换语言这操作太秀
强大又好用，已经上到项目里去了感觉还不错
期待Python后端😝就完美啦

**Comment by LanderlYoung** [comment]
哈哈，很高兴你喜欢~
笔者精力有限，欢迎大家共建~

**Comment by yqs112358** [comment]
Curious about how is the development process of Python backend 😍
@LanderlYoung

**Reply by LanderlYoung** [reply]
Busy on working, no time for this yet.
Feel free to make a PR~
